perun-core/src/main/java/cz/metacentrum/perun/core/entry/UsersManagerEntry.java

```diff
@@ -1315,21 +1317,26 @@ public void updateUserExtSourceLastAccess(PerunSession sess, UserExtSource userE
 	}
 
 	@Override
-	public List<RichUser> getSponsors(PerunSession sess, Member member, List<String> attrNames) throws PrivilegeException, UserNotExistsException {
+	public List<Sponsor> getSponsors(PerunSession sess, Member member, List<String> attrNames) throws PrivilegeException, UserNotExistsException {
```

Comment:
Because of external sites, we need to preserve the old method and also create a new one.
For entry, RPC, and open API, wee need to preserve the method returning a list of rich users (and set it as deprecated) and also create a new method (with a different name I am afraid) returning a list of sponsors instead.

Reply:
I have created a new method and annotated the old one with the `@deprecated` annotation.